vite.config.ts

```diff
@@ -10,5 +10,24 @@ export default defineConfig({
     alias: {
       '@': fileURLToPath(new URL('./src', import.meta.url))
     }
+  },
+  server: {
+    host: true,
+    port: 8080
+  },
```

Comment:
we have `vite --host` in scripts that does the same, port is just preference you can change it for yourself but doesn't need to be included in source code